manual/src/cmds/intf-c.etex

```diff
@@ -740,7 +740,9 @@ numbered from 0 to $\hbox{"string_length"}(v)-1$.
 \var{v}, with type "const char *".
 This pointer is a valid C string: there is a null byte after the last
 byte in the string. However, OCaml strings can contain embedded null bytes,
-which will confuse the usual C functions over strings.
+which will confuse the usual C functions over strings. The function
+"caml_string_is_c_safe("\var{v}")" checks if the OCaml string \var{v}
+contains any embedded null bytes.
```

Comment:
It is not obvious to me, from this description, whether the function returns a boolean or fails / fatal-errors if the function is not safe. (The name does suggest a boolean.) Maybe it could be described as `"int caml_string_is_c_safe(value "\var{v}")"`?

Reply:
> It is not obvious to me, from this description, whether the function returns a boolean or fails / fatal-errors if the function is not safe. (The name does suggest a boolean.) Maybe it could be described as `"int caml_string_is_c_safe(value "\var{v}")"`?

Good point; however, I have a slight preference for the following wording:

> the function `caml_string_is_c_safe(v)` returns a boolean indicating whether the OCaml string `v` contains any embedded null bytes.

What do you think?

Reply:
I'm perfectly happy with your suggestion, thanks!